plugin/trino-influxdb/src/main/java/io/trino/plugin/influxdb/optimize/InfluxProjectedManager.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb.optimize;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.plugin.influxdb.InfluxColumnHandle;
+import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.connector.ConnectorTableHandle;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+public class InfluxProjectedManager
+{
+    public List<String> buildProjection(ConnectorSession session, ConnectorTableHandle handle, Map<String, ColumnHandle> assignments)
```

Comment:
Move to after `InfluxMetadata.applyProjection` method. Also, remove unused arguments.